src/templates/pad.html

```diff
@@ -39,21 +39,21 @@
   <meta name="robots" content="noindex, nofollow">
   <meta name="referrer" content="no-referrer">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0">
-  <link rel="shortcut icon" href="<%=settings.faviconPad%>">
+  <link rel="shortcut icon" href="<%= staticRootAddress %>static/<%=settings.faviconPad%>">
```

Comment:
what if the faviconPad value was a URL?

Reply:
This is `ejs`, I can make a condition that separate the url and the static path.